src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -325,4 +326,88 @@ avoiding peak traffic in order to address both inconsistencies diagnostic and fi
 
 #### How to solve
 
-Under development: Task for solving mailbox inconsistencies
\ No newline at end of file
+Under development: Task for solving mailbox inconsistencies
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serves multiple applications including James, the keyspaces need isolation, 
+that also means, by the credential provided for a James server, 
+you cannot use it to access or modify other keyspaces. 
```

Comment:
```
When a Cassandra cluster is serving more than a James cluster, the keyspaces need isolation.
It can be achieved by configuring James server with credentials preventing access or modification of other keyspaces.
```